src/tester/scenarios/RandomScenarios.js

```diff
@@ -32,25 +32,29 @@ export default class RandomScenarios {
      * performing actions on random page elements
      * and trying to produce some page errors.
      * @param {Browser} instance
+     * @param {Object} testData
      * @returns {Promise<Object>} results
      */
-    async runScenario(instance) {
+    async runScenario(instance, testData) {
```

Comment:
I don't see any reason, why not to call the second argument `scenario`, since it is the very same name under which we pass it as an argument. Or we could access `startUrl` via object destructuring (`async runScenario(instance, { startUrl }) {`), since it is the only option we want.

Reply:
Argument renamed to `scenario`.